packages/synapse-react-client/src/components/SynapseTable/RowSelection/RowSelectionControls.tsx

```diff
@@ -62,6 +71,17 @@ export function RowSelectionControls(props: RowSelectionControlsProps) {
               Send to CAVATICA
             </Button>
           )}
+          {showAddToDownloadCart && (
+            <Button
+              variant="contained"
+              onClick={() => {
+                setShowDownloadConfirmation(true)
+              }}
+              startIcon={<GetAppTwoTone />}
+            >
+              Add to Download Cart
+            </Button>
+          )}
```

Comment:
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/2e830480-e705-42ea-bfd8-68c9689a3fb5)

Reply:
I thought the "X Rows Selected" was supposed to have space on either side.  Maybe a flexbox with justify-content=space-between?

2 CTAs seems to be the max that this special row selection control bar design can support.

Reply:
Adam changed that here: https://www.figma.com/file/3l3RjDnKnv8jms2XFR5BQu/Main?type=design&node-id=6332-132126&mode=dev

This is a fairly narrow screen width. We could also add icon buttons, if we need many actions